python: can't open file '/home/pvb/Analisis/SSTGlobalAnalysis/analysisDataMURDemarcaciones.py': [Errno 2] No such file or directory
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/node.py:182: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 42169 instead
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/xarray/core/indexing.py:1593: PerformanceWarning: Slicing with an out-of-order index is generating 23 times more chunks
  return self.array[key]
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 35.09 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 36.94 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/xarray/core/indexing.py:1593: PerformanceWarning: Slicing with an out-of-order index is generating 23 times more chunks
  return self.array[key]
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 17.09 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 18.93 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/xarray/core/indexing.py:1593: PerformanceWarning: Slicing with an out-of-order index is generating 23 times more chunks
  return self.array[key]
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 17.24 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 19.09 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/xarray/core/indexing.py:1593: PerformanceWarning: Slicing with an out-of-order index is generating 23 times more chunks
  return self.array[key]
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 17.61 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 19.46 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/xarray/core/indexing.py:1593: PerformanceWarning: Slicing with an out-of-order index is generating 23 times more chunks
  return self.array[key]
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 14.51 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 16.36 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/xarray/core/indexing.py:1593: PerformanceWarning: Slicing with an out-of-order index is generating 23 times more chunks
  return self.array[key]
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 14.53 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
/opt/conda/envs/ocean/lib/python3.11/site-packages/distributed/client.py:3164: UserWarning: Sending large graph of size 16.37 MiB.
This may cause some slowdown.
Consider scattering data ahead of time and using futures.
  warnings.warn(
>>>>> analysisDataMURDemarcaciones
>>>>> Cargando ficheros de /data/pvb/Satelite/MUR/NC/
>>>>> Iberian Canary Basin
>>>>> Daily 
    > Compute weigthtened mean
    > create climatology
    > Compute anomaly mean
    > Save to netcdf
    > 1803.1481852531433 seconds ---
>>>>> Demarcación marina levantino-balear
>>>>> Daily 
    > Compute weigthtened mean
    > create climatology
    > Compute anomaly mean
    > Save to netcdf
    > 621.055713891983 seconds ---
>>>>> Demarcación marina noratlántica
>>>>> Daily 
    > Compute weigthtened mean
    > create climatology
    > Compute anomaly mean
    > Save to netcdf
    > 614.7651724815369 seconds ---
>>>>> Demarcación marina canaria
>>>>> Daily 
    > Compute weigthtened mean
    > create climatology
    > Compute anomaly mean
    > Save to netcdf
    > 670.5078392028809 seconds ---
>>>>> Demarcación sudatlántica
>>>>> Daily 
    > Compute weigthtened mean
    > create climatology
    > Compute anomaly mean
    > Save to netcdf
    > 430.2421340942383 seconds ---
>>>>> Demarcación Estrecho y Alborán
>>>>> Daily 
    > Compute weigthtened mean
    > create climatology
    > Compute anomaly mean
    > Save to netcdf
    > 436.82824444770813 seconds ---
      analysisDataDemarcaciones   4672 s<<<<<
